add-gauge-report-designer.md

```diff
@@ -0,0 +1,38 @@
+---
+title: How to Add a Dynamic Gauge in Telerik Report Designer
+description: Add a Gauge to a Report with Telerik Report Designer
+type: how-to
+page_title: Add a Gauge in Telerik Report Designer
+slug: add-dynamic-gauge-report-designer
+position: 
+tags: 
+ticketid: 1473889
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td></td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+You may need to add a dynamic gauge that represents a specific value. Telerik Report Designer does not have a built-in Gauge control. You can add an image instead.   
```

Comment:
This better be stated as a question and not an assumption. For example, "I need to add a dynamic gauge that represents a specific value to my report". This will also add keywords in google and help with SEO. The rest of must go in other areas. For example, "Telerik Report Designer does not have a built-in Gauge control." must be in the **Solution** part of the KB. Then you can continue with the suggested workaround.

Reply:
Rephrased the first sentence and added a "Solution" section.